.github/workflows/trigger_prerelease.yml

```
@@ -0,0 +1,46 @@
+name: Trigger prerelease creation
+
+# This workflow triggers a prerelease creation with changelog and the release notes created by the release toolkit.
+# This workflow should be triggered merely from the default branch.
+# If you wish to be 100% free creating a prerelease, just create it manually.
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "30 10 * * 3"
+
+jobs:
+  prerelease:
+    name: Create prerelease
+    runs-on: ubuntu-latest
+    steps:
```

```suggestion
    runs-on: ubuntu-latest
    permissions:
      contents: write
    steps:
```
That allows to not have to have a token for this workflow.

---

We would have issues with the branch protection since we have checked and the actor of a job triggered by a cron job is the last one that modified the pipeline